helix-core/src/test/java/org/apache/helix/mock/MockHelixAdmin.java

```diff
@@ -45,6 +47,8 @@
 import org.apache.helix.model.ResourceConfig;
 import org.apache.helix.model.StateModelDefinition;
 import org.apache.helix.zookeeper.datamodel.ZNRecord;
+import sun.applet.Main;
```

Comment:
Remove the unused imports  in this file/PR?

Reply:
removed.